.github/workflows/build_conda_and_pypi.yaml

```
@@ -0,0 +1,58 @@
+on:
+  workflow_call:
+    inputs:
+      upload:
+        required: true
+        type: boolean
+
+jobs:
+  conda-python-build:
+    runs-on: ubuntu-latest
+    container:
+      image: condaforge/mambaforge
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          fetch-depth: 0
+      - name: Install mambabuild
+        run: mamba install -c conda-forge boa
+      - name: Build Frigate package
+        run: conda mambabuild conda/recipes/frigate --output-folder /tmp/frigate-bld
+      - name: Upload to Anaconda
+        if: ${{ inputs.upload == 'true' }}
+        run: |
+          RAPIDS_CONDA_TOKEN=${{ secrets.CONDA_RAPIDSAI_NIGHTLY_TOKEN }}
+          if [[ "${GITHUB_REF}" =~ ^refs/tags/v[0-9]{1,2}.[0-9]{2}.[0-9]{2}$ ]]; then
+            RAPIDS_CONDA_TOKEN=${{ secrets.CONDA_RAPIDSAI_TOKEN }}
+          fi
+          anaconda -t "${RAPIDS_CONDA_TOKEN}" upload --skip-existing --no-progress /tmp/frigate-bld/noarch/*.tar.bz2"
```

```suggestion
          anaconda -t "${RAPIDS_CONDA_TOKEN}" upload --skip-existing --no-progress /tmp/frigate-bld/noarch/*.tar.bz2
```

Looks like there's an extra quotation here.